src/Fmt_ast.ml

```diff
@@ -2804,13 +2860,29 @@ and fmt_cases c ctx cs =
           (fmt "@;<1000 0>")
       in
       let indent = if align_nested_match then 0 else indent in
+      let fmt_padding =
+        fmt_if_k
+          ( c.conf.align_cases
+          && not (Cmts.has_after c.cmts xlhs.ast.ppat_loc) )
+          ( match (max_len_name, pattern_len xlhs.ast) with
+          | Some max_len, Some len ->
+              let leading_len = String.length "let _ = match _ with" in
+              let case_len = max_len + String.length "|  -> " in
```

Comment:
I did this (lines 2869-2870) to avoid using magic numbers, the compiler should optimize this anyway.

Reply:
I don't think this will be optimized but the code is nice to read, don't be ashamed !